src/main/java/com/example/HR_AppJava/HR_App/Person.java

```diff
@@ -6,13 +6,18 @@ public class Person {
     private String name;
     private int age;
 
-    public void setName(){
-        System.out.println("Enter name: ");
+    public Person(){
+        setName(name);
```

Comment:
You shouldn't call your modifier methods inside of your constructor. For the default constructor, you just need 
public Person(){}
You could also add a constructor that takes parameters as
public Person(String name, int age){
this.name = name;
this.age = age;
}